docs/internal/release-process.md

```diff
@@ -0,0 +1,18 @@
+# Release process
+
+## Main package
+
+1. Create a new tag with the format `v<MAJOR>.<MINOR>.<PATCH>` depending on the
+   changes since the last release.
+
+2. Create a new GitHub release by picking the new tag and also click the
+   "Generate release notes" button to fill the release information with
+   information about the changes since the previous release.
+
+## Helm charts
```

Comment:
For charts, one has to make a change and update this version: https://github.com/grafana/flagger-k6-webhook/blob/main/charts/k6-loadtester/Chart.yaml#L5. The release tool creates the tag and release on its own